functions/go/apply-setters/README.md

```diff
@@ -4,12 +4,29 @@
 
 <!--mdtogo:Short-->
 
-Apply setter values on resource fields. Setters serve as parameters for template-free
-setting of field values.
+Update the field values parameterized by setters.
 
-Setters are a safer alternative to other substitution techniques which do not
-have the context of the structured data. Setters may be invoked to modify the
-package resources using this function to set desired values.
+##### Setters Definition
+
+Setters serve as parameters for customizing field values.
+Setters are a safer alternative to parameterize field 
+values as they have the context of the structured data.
+
+##### Setter Name: 
+
+Name of the parameter.
+
+##### Setter Value
```

Comment:
You can just use bullet points instead of sections. Also provide an example of each concept.

Reply:
Updated.